astro_2.0/features/Notifications/helpers.tsx

```diff
@@ -6,51 +6,83 @@ import {
   NotificationSettingsItem,
   NotificationSettingsPlatform,
   NotificationsGroupStatus,
+  NotificationStatus,
   NotificationType,
   NotifiedActionType,
 } from 'types/notification';
 import { NotificationSettingDTO } from 'services/NotificationsService/types';
 import uniqid from 'uniqid';
 import { DaoSettings } from 'astro_2.0/features/Notifications/types';
+import { DAO } from 'types/dao';
+import { TFunction } from 'next-i18next';
 
 export function getNotificationParamsByType(
   type: NotifiedActionType,
   daoId: string,
-  targetId: string
+  targetId: string,
+  status: NotificationStatus
 ): {
   iconType: IconName;
   url: string;
+  statusIcon: IconName | null;
 } {
   let noteType;
   let url;
+  let statusIcon: IconName | null;
 
   switch (type) {
-    case NotifiedActionType.CustomDaoCreation:
-    case NotifiedActionType.ClubDaoCreation:
-    case NotifiedActionType.FoundationDaoCreation:
-    case NotifiedActionType.CorporationDaoCreation:
-    case NotifiedActionType.CooperativeDaoCreation:
-    case NotifiedActionType.DaoNameUpdated: {
+    case NotifiedActionType.CustomDao:
+    case NotifiedActionType.ClubDao:
+    case NotifiedActionType.FoundationDao:
+    case NotifiedActionType.CorporationDao:
+    case NotifiedActionType.CooperativeDao: {
       noteType = NotificationType.DaoConfig;
       url = `/dao/${targetId}`;
 
       break;
     }
-    case NotifiedActionType.BountyDoneProposalCreation:
-    case NotifiedActionType.BountyProposalCreation: {
+
+    case NotifiedActionType.ChangePolicy:
+    case NotifiedActionType.ChangeConfig: {
+      noteType = NotificationType.DaoConfig;
+      url = `/dao/${daoId}/proposals/${targetId}`;
+
+      break;
+    }
+    case NotifiedActionType.AddMemberToRole: {
+      noteType = NotificationType.AddMember;
+      url = `/dao/${daoId}/proposals/${targetId}`;
+
+      break;
+    }
+    case NotifiedActionType.RemoveMemberFromRole: {
+      noteType = NotificationType.RemoveMember;
+      url = `/dao/${daoId}/proposals/${targetId}`;
+
+      break;
+    }
+    case NotifiedActionType.AddBounty:
+    case NotifiedActionType.BountyDone: {
       noteType = NotificationType.Bounty;
       url = `/dao/${daoId}/proposals/${targetId}`;
 
       break;
     }
 
-    case NotifiedActionType.TransferProposalCreation: {
+    case NotifiedActionType.Transfer: {
       noteType = NotificationType.Transfer;
       url = `/dao/${daoId}/proposals/${targetId}`;
 
       break;
     }
-    case NotifiedActionType.PollProposalCreation: {
+    case NotifiedActionType.FunctionCall: {
+      noteType = NotificationType.Transfer;
+      url = `/dao/${daoId}/proposals/${targetId}`;
```

Comment:
I consider we should use route constants from`routing.ts` to keep them consistent.

Reply:
fixed